### PR TITLE
Added support for mapy.cz

### PR DESCRIPTION
**Pull request by tkafka**
Mapy.cz have very nice maps of Europe and recently switched to mercator format, so I added support.

### CODE REVIEW

**Comment by tkafka** [comment]
I have rewritten the extension so that all map providers are at one place - now all you need to do is to add new entry into mapProviders array in mapjumper_content_script.js instead of editing lines of duplicate code all across the app. What do you think?

**Comment by dudarev** [comment]
@tkafka thank you so much for the pull request! I've checked mapy.cz and they seems to be quite good for Europe, similar to Yandex which we added to dev branch recently is good for Russia. Your code reorganisation looks good too. 

I'm sorry I'm a bit overloaded with work now and will have a chance to do the full test of the pull request closer to the weekend.

We are very close with completing milestone 0.6 (https://github.com/dudarev/mapjumper/milestones) two out of remaining issues there are almost done and two more are rather simple. Then we will test everything one more time in dev branch, merge to master and push to Chrome Store. Estimated time for this is end of April.

Thanks again for your contribution.

**Comment by tkafka** [comment]
@dudarev Thanks! I was about to write the exact same thing, luckily I remembered to search for already made extension first :). Let me know about the merge, so far I'll build the extension myself and send it to colleagues by mail.

**Comment by tkafka** [comment]
@dudarev I just added support for jumping directly to different map styles, it looks like this: 
screenshot 2015-04-15 17 26 58

Also, I found out fix for cases where popup doesn't resize when the script adds links.

**Comment by tkafka** [comment]
... and a refactoring to enable functions instead of just string templates for output of urls for map providers - which enables us to generate Google Earth links where the required parameter is height above Earth surface instead of zoom :).

Go from https://www.google.com/maps/@14.5524361,44.0838068,18z to https://www.google.com/maps/@14.5524361,44.0838068,512m/data=!3m1!1e3

File: src/manifest.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
             "*://www.google.co.uk/maps/*",
             "*://www.google.ru/maps/*",
             "*://www.google.com.ua/maps/*",
+            "*://www.google.cz/maps/*",
```
Comment (dudarev) [inline src/manifest.json]: There is also issue https://github.com/dudarev/mapjumper/issues/29 planned for milestone 0.7 (we are currently wrapping up 0.6) where we will add other Google TLDs as well.

**Comment by dudarev** [comment]
@tkafka Overall, everything looks very good. I'm doing some manual testing now.

I like how you put map providers into a separate file. That will allow us to write some unittests to test them specifically more cleanly. But manual testing will probably always be needed with such type of extension so I'm describing testing scenarios in https://github.com/dudarev/mapjumper/blob/master/TESTS.md

Regarding mentioning map styles in the pop up. Let's merge it to `dev`, I use `dev` version for my personal needs and can see how it feels. There is a plan to provide options where a user could select only map providers he wants, issue #21. Map styles could be added there too.
It is partially implemented in https://github.com/mityaz/mapjumper/tree/21_choose_mapping_provider_to_jump 
but not yet ready for merge. I will pick it up and try to finish by May.

One note about style: we are getting tabulations mixed up. I will probably prepare a short style guide where number of spaces will be mentioned explicitly, probably 4 spaces everywhere (now we had 4 in js, 2 in html, and now we tabs in your files). Also checking with JSHint will be one of the things there, your code is very clean, it complained only about a semicolon in one place, but for the sake of future contributors I think it should be mentioned explicitly. Added an issue for myself in the next milestone: https://github.com/dudarev/mapjumper/issues/42 So for now let's not worry about style, we always can refactore later. Please feel free to leave your thoughts in that issue.

**Comment by dudarev** [comment]
Aligning map providers to the left was a conscious design decision:

screen shot 2015-04-16 at 2 31 50 pm

This way less horizontal mouse movements are required. This is why I'm a little sceptical about adding too many map providers on a single line. 

Options feature should allow us to have less vertical movement too, especially since not all users will need some local map providers like Yandex.

**Comment by dudarev** [comment]
@tkafka your changes are merged to `dev`. Thanks a lot for them! I've made sure that all manual tests described at https://github.com/dudarev/mapjumper/blob/dev/TESTS.md pass (there were some minor issues with Foursquare). 

Also, I did change tabs to 4 spaces. As I mentioned above, there is a ticket on me to think more about common style.

Alignment is changed back to right and I also sorted everything alphabetically (vertically and horizontally), let me know your thoughts about it.

There is another pull request from @serejahh regarding jumping from Waze (issue #30). I'll incorporate it into `dev` later this week.

Milestone 0.6 is coming along, I still estimate it done and published to Chrome store end of April or beginning of May.

File: src/manifest.json (hunk 1 above)
Comment (tkafka) [inline src/manifest.json]: Great! :)

**Comment by tkafka** [comment]
@dudarev Thanks! I can see how the right alignment is useful, I was trying to quickly fulfill my needs, but you are right that it makes things less usable for 'normal user' (if such a beast exists :)) and it should be more thought out.

Great catch with the formatting, sorry for merge errors I caused. I used jshint too, but as the project doesn't have .jshintrc file now, it just accepts any indentation. Would you mind adding [.jshintrc](https://github.com/jshint/jshint/blob/master/examples/.jshintrc) file with your preferred setup?

I'll check the changes in `dev` soon, but I'm ok with everything you wrote so far anyway :).

**Pull request merged**